Networking/NetworkingTests/Responses/order-with-line-item-attributes-old-version.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
nit: add a field `note` at the same level of `data`, that explain why this is an old version

Reply:
Hmm actually I cannot add a comment to JSON ([not supported](https://stackoverflow.com/a/244858/9185596)), so I renamed the file to `order-with-line-item-attributes-before-API-support.json` and test case to `test_OrderLineItem_attributes_are_empty_before_API_support` with a more clear comment. I thought a comment in the code would be more visible :)